### PR TITLE
CUMULUS-1166 Add node README updates, change dashboard package installation to yarn

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:**

Updated docs to cover node differences between Cumulus stack and dashboard deployments.   See ticket for a better summary of the issue. 

Addresses [CUMULUS-1166](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1166)

### CODE REVIEW

File: docs/deployment/README.md [hunk 1]
```diff
@@ -346,12 +360,15 @@ If you've lost track of the needed redirect URIs, they can be located on the [AP
...
-To install the dashboard clone the Cumulus-dashboard repository into the root deploy directory and install dependencies with `npm install`:
+To install the dashboard clone the Cumulus-dashboard repository into the root deploy directory and install dependencies with `yarn install`:
 
     $ git clone https://github.com/nasa/cumulus-dashboard
     $ cd cumulus-dashboard
     $ nvm use
-    $ npm install
+    $ yarn install
+
+If you do not have the correct version of node installed, replace `nvm use` with `nvm install $(cat .nvmrc)` in the above example.
```
Comment (markdboyd) [inline docs/deployment/README.md]: I think you would want to do the `nvm install` *and* then the `nvm use`?

File: docs/deployment/README.md [hunk 2]
```diff
@@ -333,6 +333,20 @@ If you've lost track of the needed redirect URIs, they can be located on the [AP
 ----
 ## Deploy Cumulus dashboard
 
+### Dashboard Requirements
+
+Please note that the requirements are similar to the [Cumulus stack deployment requirements](#Requirements), however the node version may vary slightly and the dashboard requires yarn.    The installation instructions below include a step that will install/use the required node version referenced in the `.nvmrc` file in the dashboard repository.
```
Comment (laurenfrederick) [inline docs/deployment/README.md]: This link isn't taking me up to the requirements.

File: docs/deployment/README.md [hunk 3]
```diff
@@ -333,6 +333,20 @@ If you've lost track of the needed redirect URIs, they can be located on the [AP
...
+
+- git
+- [node 8.11.4](https://nodejs.org/en/) (use [nvm](https://github.com/creationix/nvm) to upgrade/downgrade)
+- [npm](https://www.npmjs.com/get-npm)
+- [yarn](https://yarnpkg.com/en/docs/install#mac-stable)
+- sha1sum or md5sha1sum
+- zip
+
+- AWS CLI - [AWS command line interface](https://aws.amazon.com/cli/)
+- python
```
Comment (laurenfrederick) [inline docs/deployment/README.md]: I know this isn't you, but on line 354 the 'preparing aws configuration' link doesn't work either.

File: docs/deployment/README.md (hunk 1 above)
Comment (Jkovarik) [reply docs/deployment/README.md]: The nvm install seems to  also activate the environment.

File: docs/deployment/README.md (hunk 3 above)
Comment (Jkovarik) [reply docs/deployment/README.md]: Hmm, that's  neat.  I'll see about fixing that.

Comment (Jkovarik) [reply docs/deployment/README.md]: 46df58a

File: docs/deployment/README.md (hunk 2 above)
Comment (Jkovarik) [reply docs/deployment/README.md]: 46df58a

File: docs/deployment/README.md (hunk 3 above)
Comment (Jkovarik) [reply docs/deployment/README.md]: It wasn't just that one either, the doc had several instances of that kind of link.  I fixed the deployment doc instances in the above commit.

Comment (laurenfrederick) [reply docs/deployment/README.md]: 👍 thanks!

**Pull request merged**